docs/functions/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 1
+sidebar_label: Momento Functions
+title: Momento Functions
+description: Process real-time data at massive scale with high-performance Momento Functions.
+hide_title: true
+slug: /functions
+keywords:
+  - momento
+  - functions
+  - wasm
+  - real-time processing
+  - event-driven
+  - serverless
+---
+
+# Momento Functions
+
+*Momento Functions* are a high-speed compute layer designed for processing massive amounts of real-time data. Whether you're aggregating IoT sensor data, handling financial transactions, or enriching event streams with AI, *Momento Functions* allow you to process, transform, and act on data instantly—without managing infrastructure.
+
+Built for extreme scale and performance, functions offer low-latency execution and shared global memory across all running function instances. Unlike general-purpose serverless compute, they do not execute in isolation, allowing all instances of a function to share state for efficient data processing.
```

Comment:
I think this wording isn't quite right for this implementation detail, and I'm struggling to come up with an approach that succinctly captures the differentiation's semantics. It might be best to either pull this paragraph or iterate on it more.